Arch/README.md

```diff
@@ -45,27 +45,49 @@ Arch Linux is for Users who are friendly with Linux not for beginners. if you ar
 ![](imgs/archlink.img.png)
 **After downloading do the following steps to prepare your virtual box** 
 * select **new** option
+
 ![](imgs/firstStep.img.png)
+
 * Write name of Distro like **ArchLinux**
+
 ![](imgs/distorname.img.png)
+
 * click here then choose **Other**
+  
 ![](imgs/browseISO.img.png)
+
 * choose IOS file then click **Open**
+
 ![](imgs/selectISO.img.png)
+
 * click **Next**
+
 ![](imgs/nextoption.img.png)
+
 * make **memory 2 MB** and **Processor 2 CPU **then click **Next** 
+
 ![](imgs/setMemory&CPU.img.png)
+
 * click **Next**
+
 ![](imgs/nextoption2.img.png)
+
 * click **Finish**
+
 ![](imgs/finishoption.img.png)
+
 * Click **Start** 
+
 ![](imgs/getstart.img.png)
+
 * choose the first option
+
 ![](imgs/firstpage.img.png)
+
 * then this page will apper 
```

Comment:
Typo here *appear

Reply:
I'm aware this is irrelevant to the issue at hand, but it isn't worth its own issue so we might as well fix it here